README.md

```diff
@@ -11,13 +11,40 @@ Docker distribution, ideal for beginners and studios alike.
 For development, here's what needs to happen.
 
 ```bash
-$ git clone https://github.com/getavalon/docker.git
+$ git clone https://github.com/getavalon/docker.git --recursive
 $ cd docker
 $ docker build -t avalon/docker:1.0 .
-$ docker run -ti --rm avalon/docker:1.0
+$ docker run -ti --name avalon-docker --rm avalon/docker:1.0
 ...
 ```
 
+To access and run Avalon, we will need to expose the files to the network with Samba and start the Mongo server.
+
+```bash
+$ cd docker
+$ sh samba.sh
+$ sh mongo.sh
+```
+
+You can get the IP address to the shared files with:
+
+```bash
+$ docker inspect avalon-samba | grep IPAddress
```

Comment:
Does this actually work? I would have thought it gives you the internal IP of the container, rather than the IP the Docker exposes ports through.

On Linux, the IP would simply be `localhost`. The complication on Windows is that it (may) host Docker inside a VirtualBox VM, in which case the IP would be the IP of that VM. Normally that's `192.168.99.100`.

Reply:
I'm seeing signs that this could actually work under some circumstances, like Linux and Docker for Windows, where the address is along the lines of 172.17.0.x. Did this actually work for you, @tokejepsen? Do you have Docker Toolbox or Docker for Windows installed? (If you have that little taskbar icon for Docker, then that's Docker for Windows).